src/app/src/reducers/ProfileReducer.js

```diff
@@ -135,6 +140,7 @@ export default createReducer({
             currentPassword: { $set: '' },
             newPassword: { $set: '' },
             confirmNewPassword: { $set: '' },
+            facilityLists: { $set: payload.facility_lists || [] },
```

Comment:
Consider using `|| initialState[profileSummaryFieldsEnum.facilityLists]`

Reply:
Good idea. Updated in f2d0eef